src/MediaManagerBundle/Resources/views/snippets-cms/mediaManager/detail/detail.html.twig

```diff
@@ -1,48 +1,56 @@
 {# mediaItem \ChameleonSystem\MediaManager\DataModel\MediaItemDataModel #}
 {# accessRightsMedia \ChameleonSystem\MediaManager\AccessRightsModel #}
 <div class="snippetMediaManagerDetail cms-media-item" data-id="{{ mediaItem.id }}" data-name="{{ mediaItem.name }}">
-    <div class="details">
-        <div class="image">
+    <div class="row details">
+        <div class="col-12 mb-4 col-xl-5 mb-xl-0 image mb-2">
             <img src="{{ constant('URL_MEDIA_LIBRARY') }}{{ mediaItem.path }}" alt="" class="preview">
-            <a href="{{ constant('URL_MEDIA_LIBRARY') }}{{ mediaItem.path }}" target="_blank"
-               class="btn btn-secondary"><span
-                        class="glyphicon glyphicon-download"></span> {{ 'chameleon_system_media_manager.details.download_button' | trans }}
-            </a>
+            <a href="{{ constant('URL_MEDIA_LIBRARY') }}{{ mediaItem.path }}" target="_blank" class="btn btn-success btn-sm mt-2"><i class="fas fa-download mr-2"></i>{{ 'chameleon_system_media_manager.details.download_button' | trans }}</a>
```

Comment:
Please note that lines should not be longer than 120 characters to maintain readability (doesn't need to be changed here, as it is my only comment and it's just a minor detail).